runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java

```diff
@@ -83,6 +83,11 @@
   Long getExecutionRetryDelay();
   void setExecutionRetryDelay(Long delay);
 
+  @Description("Sets the behavior of reusing objects.")
+  @Default.Boolean(false)
+  Boolean getReuseObject();
```

Comment:
I would like to keep this in line with the Flink names, i.e. `getObjectReuse()` and `setObjectReuse()`. What do you think?